static/style.css

```diff
@@ -9,7 +9,7 @@ body {
 }
 
 input[id=node]{
-    width: 222px;
+    width: 220px;
```

Comment:
Pixel tweaking... that seems extremely fragile and dependent on the fonts the user has installed. Is there no better way?